content/en/blog/_posts/2021-08-04-kubernetes-release-1.22.md

```diff
@@ -0,0 +1,157 @@
+
+For `kubeadm`, Kubernetes 1.22 also brings a new [v1beta3 configuration API](https://github.com/kubernetes/kubeadm/issues/1796). This iteration adds some long requested features and deprecates some existing ones. The v1beta3 version is now the preferred API version; the v1beta2 API also remains available and is not yet deprecated.
+
+## Major Changes
+
+### Removal of several deprecated beta APIs
+
+A number of deprecated beta APIs have been removed in 1.22 in favor of the GA version of those same APIs. All existing objects can be interacted with via stable APIs. This removal includes beta versions of the `Ingress`, `IngressClass`, `Lease`, `APIService`, `ValidatingWebhookConfiguration`, `MutatingWebhookConfiguration`, `CustomResourceDefinition`, `TokenReview`, `SubjectAccessReview`, and `CertificateSigningRequest` APIs.
+
+For the full list, check out the [Deprecated API Migration Guide](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-22) as well as the blog post [Kubernetes API and Feature Removals In 1.22: Here’s What You Need To Know](https://blog.k8s.io/2021/07/14/upcoming-changes-in-kubernetes-1-22/). 
+
+###  API changes and improvements for ephemeral containers
+
+The API used to create [Ephemeral Containers](https://kubernetes.io/docs/concepts/workloads/pods/ephemeral-containers/) changes in 1.22. The Ephemeral Containers feature is alpha and disabled by default, and the new API does not work with clients that attempt to use the old API.
+
+For stable features, the kubectl tool follows the Kubernetes [version skew policy](https://kubernetes.io/releases/version-skew-policy/); however, kubectl v1.21 and older do not support the new API for ephemeral containers. If you plan to use `kubectl debug` to create ephemeral containers, and your cluster is running Kubernetes v1.22, you cannot do so with kubectl v1.21 or earlier. Please update kubectl to 1.22 if you wish to use `kubectl debug` with a mix of cluster versions.
+
+## Other Updates
+
+### Graduated to Stable
+
+* [Bound Service Account Token Volumes](https://github.com/kubernetes/enhancements/issues/542)
+* [CSI Service Account Token](https://github.com/kubernetes/enhancements/issues/2047)
+* [Windows Support for CSI Plugins](https://github.com/kubernetes/enhancements/issues/1122)
+* [Warning mechanism for deprecated API use](https://github.com/kubernetes/enhancements/issues/1693)
+* [PodDisruptionBudget Eviction](https://github.com/kubernetes/enhancements/issues/85)
+* [CronJobs](https://github.com/kubernetes/enhancements/issues/19)
+
+### Notable Feature Updates
+
+* [PodSecurity admission](https://github.com/kubernetes/enhancements/issues/2579) (PodSecurityPolicy replacement)
+* [Memory Manager](https://github.com/kubernetes/enhancements/issues/1769)
```

Comment:
```suggestion
* The [Memory Manager](https://github.com/kubernetes/enhancements/issues/1769)
  moves to beta
```
(I'd prefer to hyperlink all of these feature updates to the relevant docs rather than to KEPs - what do people think?)

Reply:
re: 'moves to beta', we actually had that for all of these. i'll go back and put them in as i think it makes more sense.

re: linking docs vs KEPs, i think we should consider that. this blog follows past practice (at least going back three versions). are you passionate about discussing a change for this release, or would it be ok to add this to 1.23 considerations?

Reply:
> re: linking docs vs KEPs, i think we should consider that. this blog follows past practice (at least going back three versions). are you passionate about discussing a change for this release, or would it be ok to add this to 1.23 considerations?

It's fine to park that detail.